app/src/main/res/xml-v25/shortcuts.xml

```diff
@@ -0,0 +1,22 @@
+<shortcuts xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <shortcut
+        android:shortcutId="shortcut_new_post"
+        android:enabled="true"
+        android:icon="@drawable/qc_post_plus"
```

Comment:
Please use vector drawables; you can import a svg via android studio very easy with option below the normal import option